pywbem/_subscription_manager.py

```diff
@@ -1253,29 +1276,31 @@ def _create_filter(self, server_id, source_namespace, query,
         # Validate server_id
         server = self._get_server(server_id)
 
-        this_host = getfqdn()
         ownership = "owned" if owned else "permanent"
 
-        filter_path = CIMInstanceName(FILTER_CLASSNAME,
-                                      namespace=server.interop_ns)
+        svr_name = self._get_systemname(server_id)
```

Comment:
Can we call this variable "system_name" instead?

Reply:
See above  DONE.